CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All changes will be logged here
 
+## 0.4.1 [2020-06-08]
```

Comment:
should be `0.5.0`. See [Semantic Versioning](https://semver.org/) Also need version bump in `package.json`.

Reply:
Got it!